src/Debug.c

```diff
@@ -36,6 +36,7 @@ void Ros_Debug_Init()
 void Ros_Debug_BroadcastMsg(char* fmt, ...)
 {
     char str[MAX_DEBUG_MESSAGE_SIZE];
+    char Formatted_Time[300];
```

Comment:
Please also use [snake_case](https://en.wikipedia.org/wiki/Snake_case) for variable names.

Reply:
Will make sure all the variables are in that format now and for future use as well.